com.unity.multiplayer.mlapi/Runtime/Core/MessageBatcher.cs

```diff
@@ -185,7 +189,7 @@ public void SendItems(int threshold, SendCallbackType sendCallback)
         /// <param name="messageType"> the message type to pass back to callback</param>
         /// <param name="clientId"> the clientId to pass back to callback</param>
         /// <param name="receiveTime"> the packet receive time to pass back to callback</param>
-        public int ReceiveItems(in BitStream messageStream, ReceiveCallbackType receiveCallback, MLAPI.RpcQueueContainer.QueueItemType messageType, ulong clientId, float receiveTime)
+        public int ReceiveItems(in BitStream messageStream, ReceiveCallbackType receiveCallback, RpcQueueContainer.QueueItemType messageType, ulong clientId, float receiveTime)
         {
             using PooledBitStream copy = PooledBitStream.Get();
```

Comment:
this is C# 8.0 — I'd suggest `using (var copy = PooledBitStream.Get()) { ... }` block syntax instead!

Reply:
(in fact, I'm making this change in a different PR so feel free to leave it up to me)